les/handler.go

```diff
@@ -1040,6 +1069,25 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 
 		p.fcServer.ReceivedReply(resp.ReqID, resp.BV)
 
+	case StopMsg:
+		if pm.odr == nil {
+			return errResp(ErrUnexpectedResponse, "")
+		}
+		p.freezeServer(true)
+		pm.retriever.frozen(p)
+		p.Log().Warn("Service stopped")
+
+	case ResumeMsg:
+		if pm.odr == nil {
+			return errResp(ErrUnexpectedResponse, "")
+		}
+		var bv uint64
```

Comment:
bv is not used here.

Reply:
Right, I forgot to implement this part :) It does work like this but I wanted to update the buffer status too when resuming normal operation. It is being updated now.